### PR TITLE
Update oppm.lua to make it install itself from loot disk

### PR DESCRIPTION
**Pull request by Vexatos**
Too many reports of OPPM not working due to the floppy being read-only.

### CODE REVIEW

**Comment by Vexatos** [comment]
iKnowWhatIAmDoing. :wink:

File: src/main/resources/assets/opencomputers/loot/OPPM/oppm.lua [hunk 1]
```diff
@@ -480,18 +480,82 @@ local function updatePackage(pack)
...
+  local number = tonumber(result)
+  if number and number > 0 and number <= #candidates then
+    choice = candidates[number]
+  else
+    print("Invalid input, please try again.")
+  end
+end
+candidates = nil
+
+print("Installing OpenOS to device " .. (choice.getLabel() or choice.address))
```
Comment (rmellema) [inline src/main/resources/assets/opencomputers/loot/OPPM/oppm.lua:547]: It is so much stolen from Sangar's `install.lua` that it still wants to install OpenOS :P

Comment (Vexatos) [inline src/main/resources/assets/opencomputers/loot/OPPM/oppm.lua:547]: Yes, yes it is.

Comment (fnuecke) [inline src/main/resources/assets/opencomputers/loot/OPPM/oppm.lua:547]: Haha, nice one. I'll adjust it to read "Installing pirated software to device", give me a moment.

**Pull request merged**